src/components/tabs/__internal__/tabs-header/tabs-header.style.ts

```diff
@@ -25,6 +25,11 @@ const StyledTabsHeaderWrapper = styled.div<StyledTabsHeaderWrapperProps>`
       box-sizing: border-box;
       padding: 3px;
 
+      button[data-element="select-tab"] {
+        border-top-left-radius: var(--borderRadius100);
```

Comment:
The top left corner radius is already applied in `tab-title.style` so this line isn't needed

Reply:
The pointed out line does apply styles in few places.